README.adoc

```diff
@@ -0,0 +1,127 @@
+= The Agriculture Core
+
+_The Agriculture Core_ is a NodeJS server exposing a robust REST API for various client applications to connect to.
+
+== Configuration
+
+The agriculture-core pulls its configuration from a `.env` file at the root of the project.
+These configuration variables are loaded into `process.env.<VAR_NAME>` and are accessible accross the application.
+To get started, this configuration file must be created with the following format:
+
+[source,properties]
+----
+# Application Configuration
+PORT=
+
+# Database Environment
+DB_HOST=
+DB_NAME=
+DB_USER=
+DB_PASS=
+
+# Authentication
+JWT_SECRET=
+JWT_ISSUER=
+JWT_AUDIENCE=
+JWT_EXPIRES=
+
+# Logging
+LOG_LEVEL=
+----
+
+`LOG_LEVEL` can be configured from levels 0-5:
+
+ * error: 0
+ * warn: 1
+ * info: 2
+ * verbose: 3
+ * debug: 4
+ * silly: 5
+
+== Development
+
+=== Dependencies
+
+* link:https://nodejs.org/en/[NodeJS] is the framework in use.
+* link:https://yarnpkg.com/lang/en/docs/install/[yarn] handles all other dependencies.
+* MySQL server indicated by the `.env` file.
+
+To install all other dependencies with use the command `yarn`.
+
+NOTE: The `knex` configuration must also be altered to choose the right `client`.
```

Comment:
Just mentioning this note is still here, if that was intentional that's fine :D

Reply:
Yup intentional